hapi/mixins/threading.py

```diff
@@ -74,19 +75,19 @@ def _create_curl(self, url, headers, data):
         c.data = data
 
         c.status = -1
-        c.body = cStringIO.StringIO()
-        c.response_headers = cStringIO.StringIO()
+        c.body = io.StringIO()
```

Comment:
I suspect that these really should be `six.BytesIO` -- http headers are bytes, not unicode

Reply:
Done.